skiko/src/commonMain/kotlin/org/jetbrains/skia/Typeface.kt

```diff
@@ -100,7 +110,18 @@ class Typeface internal constructor(ptr: NativePointer) : RefCnt(ptr) {
     val variationAxes: Array<FontVariationAxis>?
         get() = try {
             Stats.onNativeCall()
-            _nGetVariationAxes(_ptr)
+            val axisCount = _nGetVariationAxesCount(_ptr)
+            if (axisCount == 0) {
```

Comment:
This can be -1 if error occurred, see [Skia docs](https://api.skia.org/classSkTypeface.html#a6afe4ed9b3c3e1d0ebdb210805f51c68)